manifests/n/NielsenKellerman/LiNK/1.32/NielsenKellerman.LiNK.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created with YamlCreate.ps1 v2.2.3 $debug=NVS1.CRLF.7-3-3.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: NielsenKellerman.LiNK
+PackageVersion: "1.32"
+InstallerType: inno
+UpgradeBehavior: install
+ReleaseDate: 2018-11-13
+Installers:
+- Architecture: x86
+  InstallerUrl: https://nksports.com/mwdownloads/download/link/id/186
+  InstallerSha256: 6f7068f912f72a9fc0df320a57be9545e9eea03e41eb12c24dc8347c4eda8aae
```

Comment:
```suggestion
  InstallerSha256: 833282B35BBD320418D1C597CE3748AC7F2ED56DBBBA574FF7B2CF1EF96D4BCD
```

Reply:
I'm still seeing `6f7068f912f72a9fc0df320a57be9545e9eea03e41eb12c24dc8347c4eda8aae` as the hash for the file. Could this be a CDN thing?

![image](https://user-images.githubusercontent.com/12696894/226062950-441ab9c9-f2f5-4830-bc66-f43e03985868.png)

Reply:
Hey @stephengillie - the hash mismatch is occurring because the site seems to occasionally serve an error page rather than the exe file. The hash I have provided matches the exe file, but the CI is failing because it is downloading the HTML of the error page. I have attached this html in my [previous comment](https://github.com/microsoft/winget-pkgs/pull/99354#issuecomment-1467108810), but I'm not sure how to get around this.

Reply:
Hi @HandyHat,

It appears that the developer's host doesn't support automated downloads. There's unfortunately a possibility that this package might not be a good fit for addition to the public manifests. 
![image](https://user-images.githubusercontent.com/114952565/226466629-c693e32c-0b76-44c0-bbc2-dfc2cae233b3.png)

Reply:
Good catch, @stephengillie. That is indeed the case. Just tried it myself by trying to download it through the terminal and it throws an error.